python/lsst/pipe/tasks/selectImages.py

```diff
@@ -315,6 +315,13 @@ class PsfWcsSelectImagesConfig(pipeBase.PipelineTaskConfig,
         deprecated=('This field has been moved to ComputeExposureSummaryStatsTask and '
                     'will be removed after v24.')
     )
+    doLegacyStarSelectionComputation = pexConfig.Field(
+        doc="Perform the legacy star selection computations (for backwards compatibility)",
+        dtype=bool,
+        default=False,
```

Comment:
About the default value, should this be `True` by default making the users of `visitSummaryTable` use this config?

Reply:
So this should only be set if you are in the case that you are running new coadd code on old calexp data, which I assume is not the norm.
Furthermore, this is an odd case (and you can complain!) that this is a new deprecated function (!!!) but in this way when somebody overrides it they will get a warning, and that will let them know that they are running in a specific backwards compatibility mode.

Reply:
Fair enough!